libs/dkg/include/dkg/rbc_messages.hpp

```diff
@@ -0,0 +1,253 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "core/byte_array/const_byte_array.hpp"
+#include "core/serializers/byte_array_buffer.hpp"
+#include "core/serializers/stl_types.hpp"
+
+#include <cstdint>
+
+namespace fetch {
+namespace dkg {
+namespace rbc {
+
+using TruncatedHash        = byte_array::ByteArray;
```

Comment:
We usually put these aliases in a class since any file including rpc_messages.hpp will see these aliases which can cause conflicts sometimes. It's ok in cpp files since these are never included.